src/conversion/buildmodelconverter.js

```diff
@@ -15,8 +15,8 @@ import {
 	removeUIElement,
 	wrapItem,
 	unwrapItem,
-	convertTextsInsideMarker,
-	convertElementsInsideMarker
+	highlightTexts,
+	highlightElements
```

Comment:
All other methods here are singular (`wrapItem`, `removeUIElement`). I think this should be too.